lib/slack_bot_wrapper.ex

```diff
@@ -5,18 +5,9 @@ defmodule SlackBotWrapper do
     {:ok, state}
   end
 
-  def format_status(_reason, [_pdict, state]) do
-    [data: [{'State', "My current state is '#{inspect(state)}', and I'm happy"}]]
-  end
-
   def start_link(token) do
-    Logger.info(fn -> "Starting #{__MODULE__} with #{token}" end)
-
-    if token do
-      Slack.Bot.start_link(SlackEventHandler, [], token)
-    else
-      GenServer.start_link(__MODULE__, [], name: __MODULE__)
-    end
+    Logger.info(fn -> "Starting #{__MODULE__}" end)
+    Slack.Bot.start_link(SlackEventHandler, [], token)
```

Comment:
At this point, we probably don't even need `SlackBotWrapper` because we could do the conditional to start run this `Slack.Bot.start_link` method in the App.  But I don't mind the layer between us and the libraries we use.